### DISCUSSION TITLE
Does ServiceMonitor support examplars?

### DISCUSSION THREAD

**Discussion by caibirdme** [prometheus-operator/prometheus-operator#5178] [Q&A]
The App instruments metrics with examplars, and I can get those examplar by running `curl -H 'accept: xxxx' ip:port/metrics` .
It returns something like:
```
rpc_server_handled_total{app="test"} 55.0 # {traceID="17b264bff45ff2f031a997681b5fd0d6"} 1.0 1.669020032149943e+09
```
But when I search on the grafana and open the examplar button
```
sum(increase(rpc_server_handled_total[1m]))
```
examplars don't appear. Am I missing something?

**Comment by simonpasquier** [comment]
If you're using vanilla Prometheus with the operator, you probably need to enable the exemplars storage by setting `spec.exemplars: {}` in the Prometheus resource.
